src/index.js

```diff
@@ -122,6 +122,15 @@ class FreeSurfer {
 
 }
 FreeSurfer.prototype.schema = schema.schema;
+
+/**
+ * Valid fields.
+ * @static
+ *
+ * @const {string[]}
+ */
+FreeSurfer.validFields = schema.validFields;
```

Comment:
This merely exports the schema (valid FreeSurfer features) so they’re usable in MRN-Code/decentralized-laplacian-ridge-regression as inputs.